guide/src/module.md

```diff
@@ -48,53 +53,110 @@ To import the module, either:
 
 ## Documentation
 
-The [Rust doc comments](https://doc.rust-lang.org/stable/book/first-edition/comments.html) of the module
+The [Rust doc comments](https://doc.rust-lang.org/stable/book/ch03-04-comments.html) of the module
 initialization function will be applied automatically as the Python docstring of your module.
 
+For example, building off of the above code, this will print `This module is implemented in Rust.`:
+
 ```python
-import rust2py
+import my_extension
 
-print(rust2py.__doc__)
+print(my_extension.__doc__)
 ```
 
-Which means that the above Python code will print `This module is implemented in Rust.`.
+## Organizing your module registration code
+
+As your Python module grows, it can be helpful to split your Rust code into several Rust modules to
+keep your code readable.
+
+You can pass references to the `PyModule` so that each 
+module registers its own FFI code, which can give your project better 
+[encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)). For example:
+
+```rust,ignore
```

Comment:
I'd like to not ignore this, but don't know how to simulate multiple distinct files in a doc test.

Reply:
You can do https://doc.rust-lang.org/rustdoc/documentation-tests.html#hiding-portions-of-the-example in rustdoc. I'm not sure if this also works in mdbook (?)

Reply:
It does. You could use hidden `# mod lib { }` etc to break the code into multiple "files" (modules).